presentation/src/main/java/com/android254/presentation/about/view/AboutScreen.kt

```diff
@@ -15,20 +15,294 @@
  */
 package com.android254.presentation.about.view
 
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.*
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.android254.presentation.common.theme.DroidconKE2022Theme
 
 @Composable
 fun AboutScreen() {
-    Text(text = "About Screen")
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(MaterialTheme.colorScheme.background)
+            .padding(top = 13.dp, bottom = 45.dp)
+            .verticalScroll(rememberScrollState()),
+        verticalArrangement = Arrangement.spacedBy(16.dp)
+    ) {
+
+        aboutHeaderComponent()
+        Image(
+            Icons.Filled.PropaneTank,
+            contentDescription = "About image",
+            contentScale = ContentScale.Crop,
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(235.dp)
+        )
+
+        aboutBody()
+
+    }
+}
+
+
+@Composable
+fun aboutBody() {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(start = 13.dp, end = 5.dp),
+        verticalArrangement = Arrangement.spacedBy(23.dp)
+    ) {
+
+
+        aboutComponent()
+        organizingTeamComponent()
+        organisedByComponent()
+
+
+    }
+}
+
+@Composable
+fun organisedByComponent() {
```

Comment:
move separate components to `com.android254.presentation.about.components` package. To ensure small files.

Reply:
I updated this